### DISCUSSION TITLE
DREAM3D file cannot be read by DAMASK after EBSD reconstruction

### DISCUSSION THREAD

**Discussion by TsingYi11** [BlueQuartzSoftware/DREAM3DNX-Issues#22] [Q&A]
DREAM3D file cannot be read by DAMASK after EBSD reconstruction
![15ca89a9f690d832740b3bc8c9fba78](https://github.com/user-attachments/assets/7f07f1fe-5dd6-4eae-b599-c2c0ee66523b)

**Comment by imikejackson** [comment]
@TsingYi11 
I am assuming this is a DREAM3D-NX file? If so there were changes that happened to the internal structure of the .dream3d file. The DAMASK import utilities will need an update to be able to read the data. Do you know who we can contact at DAMASK to help them with an update?

**Reply by TsingYi11** [reply]
Hello, Mike; as you know, DAMASK was developed by Max-Planck-Institut für Eisenforschung GmbH(MPIE), so you'd better contact its head, Franz Roters, who also discussed your speech in dream3d. It's easy for you to find his information. I attached the link to the official website; I hope it helps you.(https://mpie.de/4981753/damask-3)
